test/coverage/communication/PublisherTest.cpp

```diff
@@ -11,30 +11,93 @@
 
 #include <gtest/gtest.h>
 #include <up-cpp/communication/Publisher.h>
+#include <up-cpp/datamodel/builder/Payload.h>
+#include <up-cpp/datamodel/serializer/UUri.h>
+#include <uprotocol/v1/uri.pb.h>
 
 #include "UTransportMock.h"
+#include "up-cpp/communication/Publisher.h"
+#include "up-cpp/datamodel/builder/UMessage.h"
+#include "up-cpp/transport/UTransport.h"
 
+using namespace uprotocol::communication;
+using namespace uprotocol::datamodel::builder;
+using namespace uprotocol::v1;
+using ::testing::_;
+using ::testing::Return;
 namespace {
+using namespace uprotocol::datamodel::builder;
 
-class TestFixture : public testing::Test {
+class TestPublisher : public testing::Test {
 protected:
 	// Run once per TEST_F.
 	// Used to set up clean environments per test.
-	void SetUp() override {}
+	void SetUp() override {
+		source_.set_authority_name("10.0.0.1");
+		source_.set_ue_id(0x00011101);
+		source_.set_ue_version_major(0xF1);
+		source_.set_resource_id(0x0);
+
+		topic_.set_authority_name("10.0.0.1");
+		topic_.set_ue_id(0x00011101);
+		topic_.set_ue_version_major(0xF8);
+		topic_.set_resource_id(0x8101);
+		transportMock_ =
+		    std::make_shared<uprotocol::test::UTransportMock>(source_);
+
+		format_ = UPayloadFormat::UPAYLOAD_FORMAT_TEXT;
+		priority_ = UPriority::UPRIORITY_CS1;
```

Comment:
The code uses this as the default value when no priority is specified. Set this to something else (maybe CS2)

Reply:
done